pkg/saga/statemachine/store/repository/state_machine_repository.go

```diff
@@ -205,7 +204,7 @@ func (s *StateMachineRepositoryImpl) RegistryStateMachineByReader(reader io.Read
 	return nil
 }
 
-func (s *StateMachineRepositoryImpl) SetStateLangStore(stateLangStore *db.StateLangStore) {
+func (s *StateMachineRepositoryImpl) SetStateLangStore(stateLangStore core.StateLangStore) {
```

Comment:
Does the caller also need to be modified?

Reply:
I think it is necessary. If you specify a type other than the interface, it may be limited to the caller's use. Now adjusting to the type of interface will not affect the user's call.